tests/Admin/AdminTest.php

```diff
@@ -2282,32 +2282,18 @@ public function testGetBatchActions(): void
     }
 
     /**
-     * @covers \Sonata\AdminBundle\Admin\AbstractAdmin::showMosaicButton
+     * @covers \Sonata\AdminBundle\Admin\AbstractAdmin::setListModes
```

Comment:
@sonata-project/contributors Do we use @cover annotation ?

Reply:
This suit test is full of `covers`, i just *mimic* 

https://github.com/sonata-project/SonataAdminBundle/blob/d326a58f75608e4cb2d242439ffc3f3599281664/tests/Admin/AdminTest.php